bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeperAdmin.java

```diff
@@ -148,13 +149,13 @@ public BookKeeperAdmin(ClientConfiguration conf) throws IOException, Interrupted
      * @param bkc
      *            - bookkeeper instance
      */
-    public BookKeeperAdmin(final BookKeeper bkc) {
+    public BookKeeperAdmin(final BookKeeper bkc, StatsLogger statsLogger) {
```

Comment:
Sure, I'll leave the old constructor. I should have noticed that it was defined public...